README.md

```diff
@@ -190,7 +290,11 @@ To create 200 cases in various states with various responders for search testing
 ```
 rake seed:search:data
 ```
-
+In order for this task to run sucessfully, redis needs to be running
+```
+redis-server
+```
+and AWS access needs to have been setup, as creating cases (even on development) uploads file to S3
```

Comment:
Do we need to provide details on how to get your AWS account?

Reply:
we could do, but I don't actually know how mine happened?